explore/src/main/scala/explore/tabs/ObsTabContents.scala

```diff
@@ -392,21 +392,24 @@ object ObsTabContents {
                 .zoom(atMapWithDefault(obsId, UndoStacks.empty))
             )
 
-          TileController(
-            props.userId.get,
-            coreWidth,
-            defaultLayouts,
-            layouts,
-            List(
-              targetTile,
-              notesTile,
-              skyPlotTile,
-              constraintsTile,
-              configurationTile
-            ),
-            GridLayoutSection.ObservationsLayout,
-            clazz = ExploreStyles.ObservationTiles.some
-          ): VdomNode
+          val rglRender: LayoutsMap => VdomNode = (l: LayoutsMap) =>
+            TileController(
+              props.userId.get,
+              coreWidth,
+              defaultLayouts,
+              l,
+              List(
+                targetTile,
+                notesTile,
+                skyPlotTile,
+                constraintsTile,
+                configurationTile
+              ),
+              GridLayoutSection.ObservationsLayout,
+              clazz = ExploreStyles.ObservationTiles.some
+            )
+
+          potRender[LayoutsMap](rglRender.reuseAlways)(layouts.get)
```

Comment:
Does this rerender when the values that `rglRender` uses change? `props.userId`, `coreWidth`, the tiles, etc? Otherwise, we may want to make reusability depend on those.

Reply:
Excellent question. I'm not sure, This is inside another `Reuse.by` at line
https://github.com/gemini-hlsw/explore/blob/master/explore/src/main/scala/explore/tabs/ObsTabContents.scala#L305=

Do we need it twice? I get lost with nested reusabilities